arangod/RestServer/FlushFeature.cpp

```diff
@@ -518,9 +518,14 @@ class RocksDBRecoveryHelper final: public arangodb::RocksDBRecoveryHelper {
 };
 
 void registerRecoveryHelper() {
+  static bool done(false);
   static const MMFilesRecoveryHelper mmfilesHelper;
   static const RocksDBRecoveryHelper rocksDBHelper;
 
+  if (done) {
```

Comment:
how is this possible, are features prepared more than once?

Reply:
in tests they are, that is how I stumbled onto this